docs/rules/require-await.md

```diff
@@ -47,7 +64,22 @@ async function noop() {}
 
 ## When Not To Use It
 
-If you don't want to notify async functions which have no `await` expression, then it's safe to disable this rule.
+Asynchronous functions are designed to work with promises such that throwing an error will cause a promise's rejection handler (such as `catch()`) to be called. For example:
+
+```js
+async function fail() {
+    throw new Error("Failure!");
+}
+
+fail().catch(error => {
+    console.log(error.message);
+});
+```
+
+In this case, the `fail()` function throws an error that is intended to be caught the `catch()` handler assigned later. Converting the `fail()` function into an synchronous function would require the call to `fail()` to be refactored to use a `try-catch` statement instead of a promise.
```

Comment:
Missing preposition: "[...] that is intended to be caught [in?] the `catch()` handler assigned later."

Reply:
Fixed.